.eslintignore

```diff
@@ -0,0 +1 @@
+tasks
```

Comment:
Instead of creating a new file.. let's add it to the `eslintrc.json`

`"ignorePatterns": ["tasks"],`

https://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories

Reply:
👍

Reply:
updated